dbscripts/db-init.sql

```diff
@@ -208,8 +208,10 @@ CREATE TABLE ProductPayments (
 -- ProductExports
 
 CREATE TABLE ProductExports (
```

Comment:
don't we need to track who exported the product? ie. which trader is adding it?

Reply:
This wasn't what was originally agreed to. Is this something we definitely want?

Reply:
recorderUuid